glide.yaml

```diff
@@ -17,3 +17,4 @@ import:
   version: 13fc8573324cd2af42c88043b5a7c6e03bf39f95
   repo: https://github.com/kubevirt/qe-tools.git
 - package: github.com/spf13/pflag
+- package: github.com/docker/spdystream
```

Comment:
what is this dependency used for? I don't see it imported anywhere in our code.

Reply:
exec_util.go uses it to open the stream to the container.

Reply:
we aren't importing the docker package anywhere in our code though. This package is only used within the k8s package that's responsible for making the remote connection.

With glide, we don't manage our dependencies' dependencies. That's supposed to be taken care of for us.

try removing this dependency from glide.yaml and running ```glide cc && glide update --strip-vendor``` again. I'd hope that docker dependency get's pulled in. That works in KubeVirt. We have to do the same thing.